qiskit_bot/api.py

```diff
@@ -149,7 +149,16 @@ def on_pull_event(data):
     if data['action'] == 'opened':
         repo_name = data['repository']['full_name']
         pr_number = data['pull_request']['number']
+        pr = REPOS[repo_name].gh_repo.get_pull(pr_number)
         if repo_name in REPOS:
+            if data['pull_request']['author_association'] != 'MEMBER':
+                # tag PR with 'community PR' label & notify reviewers
+                labels = pr.get_labels()
+                label_names = [label.name for label in labels]
+                if "Community PR" not in label_names:
+                    pr.add_to_labels("Community PR")
+                    pr.create_review_request(
+                        team_reviewers="community-reviewers")
```

Comment:
I think it would be good to split this off into a self contained function. Mostly to try and limit the amount of code that does functional things in the `api.py` module. Since as the level of functionality grows tracking through too much logic in here will be tricky. But if we made the pr event handler function basically just:

```
if data['action'] == 'opened':
     repo_name = data['repository']['full_name']
     pr_number = data['pull_request']['number']
     if repo_name in REPOS:
          notification.process_community_review(pr_number, REPOS[repo_name], CONFIG)
          notifications.trigger_notifications(pr_number, REPOS[repo_name], CONFIG)
```

As we expand the things we're doing we can just add them as standalone functions. This also makes it easier to add more async handling of events as needed. It's probably not an issue for this addition since the processing is going to be quick and we're not doing too much on open PR events. But having the flexibility to handle `trigger_notifications` and the community review piece in parallel async may be necessary as we add more features.